examples/launch_ball_from_client.py

```diff
@@ -8,7 +9,8 @@
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(
-        description="Start client, send set state request and then launch ball request to ball launcher server."
+        description="Start client, send set state request and then launch ball request"
+        "to ball launcher server."
```

Comment:
Just a tip (may not make sense in all situations): I typically use `argparse.ArgumentParser(description=__doc__)` to simply re-use the docstring of the file for the help.

Reply:
Ahh, cool, did not know that! Is changed in the newest commit!